news/models.py

```diff
@@ -177,17 +185,54 @@ def copy_relations(self, oldinstance):
         """
         self.tags.set(oldinstance.tags.all())
 
-    def get_latest_articles(self, limit=3):
+    @property
+    def category_articles(self):
+        """
+        Return a queryset for a category
+        """
+        return (
```

Comment:
Is it Black that adds the parenthises around this?

Reply:
Yeah. It's funky about having a queryset like this split over multiple lines

Reply:
Thought as much